types/insane/index.d.ts

```diff
@@ -0,0 +1,83 @@
+// Type definitions for insane 1.0.0
+
+type AllowedSchemes = "http" | "https" | "mailto";
+
+type AllowedTags =
+    | "a"
+    | "abbr"
+    | "article"
+    | "b"
+    | "blockquote"
+    | "br"
+    | "caption"
+    | "code"
+    | "del"
+    | "details"
+    | "div"
+    | "em"
+    | "h1"
+    | "h2"
+    | "h3"
+    | "h4"
+    | "h5"
+    | "h6"
+    | "hr"
+    | "i"
+    | "img"
+    | "ins"
+    | "kbd"
+    | "li"
+    | "main"
+    | "ol"
+    | "p"
+    | "pre"
+    | "section"
+    | "span"
+    | "strong"
+    | "sub"
+    | "summary"
+    | "sup"
+    | "table"
+    | "tbody"
+    | "td"
+    | "th"
+    | "thead"
+    | "tr"
+    | "u"
+    | "ul";
+
+type EmptyObject = Record<string, never>;
+
+type AllowedAttributes = Record<AllowedTags, string[] | never[]>;
+
+type AllowedClasses = Record<AllowedTags, string[]> | EmptyObject;
+
```

Comment:
Not a correct definition per https://github.com/bevacqua/insane?tab=readme-ov-file#example-1

You should *at a minimum* have tests that are whatever is in the project README

Reply:
Done